<?xml version="1.0" encoding="UTF-8"?>
<testng-results total="3" passed="2" failed="0" skipped="1">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-08-18T11:31:13Z" name="Default suite" finished-at="2021-08-18T11:31:13Z" duration-ms="47">
    <groups>
    </groups>
    <test started-at="2021-08-18T11:31:13Z" name="Default test" finished-at="2021-08-18T11:31:13Z" duration-ms="47">
      <class name="com.Day22.Skip.SkipTest">
        <test-method signature="Test1()[pri:0, instance:com.Day22.Skip.SkipTest@2794eab6]" started-at="2021-08-18T17:01:13Z" name="Test1" finished-at="2021-08-18T17:01:13Z" duration-ms="14" status="SKIP">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping Test 1 as flag is false.]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping Test 1 as flag is false.
	at com.Day22.Skip.SkipTest.Test1(SkipTest.java:19)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Test1 -->
        <test-method signature="Test2()[pri:0, instance:com.Day22.Skip.SkipTest@2794eab6]" started-at="2021-08-18T17:01:13Z" name="Test2" finished-at="2021-08-18T17:01:13Z" duration-ms="3" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Test2 -->
        <test-method signature="Test3()[pri:0, instance:com.Day22.Skip.SkipTest@2794eab6]" started-at="2021-08-18T17:01:13Z" name="Test3" finished-at="2021-08-18T17:01:13Z" duration-ms="2" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Test3 -->
      </class> <!-- com.Day22.Skip.SkipTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
